proto/vtrpc.proto

```diff
@@ -33,8 +33,55 @@ message CallerID {
   string subcomponent = 3;
 }
 
-// ErrorCode is the enum values for Errors
+// ErrorCode is the enum values for Errors. Internally, errors should
+// be created with one of these codes. These will then be translated over the wire
+// by various RPC frameworks.
 enum ErrorCode {
+  SUCCESS = 0;
+  // Context was cancelled (and noticed in the app layer, as opposed to the RPC layer)
+  CANCELLED = 1;
+  // MySQL error code that we don't explicitly handle
+  MYSQL_ERROR = 2;
```

Comment:
See comment on design doc, we need one value per canonical error code in RPC framework, not more...